consensus/testing.go

```diff
@@ -26,40 +21,63 @@ func RequireNewTipSet(require *require.Assertions, blks ...*types.Block) types.T
 	return ts
 }
 
-// TestPowerTableView is an implementation of the powertable view used for testing mining
-// wherein each miner has totalPower/minerPower power.
-type TestPowerTableView struct {
-	minerPower, totalPower *types.BytesAmount
-	minerToWorker          map[address.Address]address.Address
+// TestActorState provides a queryer that responds to power table view queries with the given parameters
+type TestActorState struct {
```

Comment:
Nit: because `Test` is necessarily used as a verb in Go, I find structures called TestX to cause friction in my head. It's also a little bit too much on the side of a component telling the user how it should be used, rather than what it provides.

`FakeActorState` is unambiguous and unopinionated.